README.md

```diff
@@ -57,36 +51,47 @@ Package Name | Minimum Version
 
 Package Name | Minimum Version
 ------------ |:--------------:
-[sphinx](http://www.sphinx-doc.org/) | 1.4
-[inkscape](http://inkscape.org) |
+[`sphinx`](http://www.sphinx-doc.org/) | 1.8
 
-> **Note:** We recommend using [pip](https://pypi.org/project/pip) to manage Python modules.
```

Comment:
Any particular reason for removing this?

Reply:
Just as we don't suggest how people manage any of the other tools...